csp/tests/test_engine.py

```diff
@@ -1817,15 +1818,13 @@ def test_graph_shape_bug(self):
 
         @csp.graph
         def aux(x: [ts[float]], y: {str: ts[float]}) -> csp.Outputs(
-            o1=csp.OutputBasket(typing.List[ts[float]], shape_of="x"),
+            o1=csp.OutputBasket(List[ts[float]], shape_of="x"),
             o2=csp.OutputBasket(typing.Dict[str, ts[float]], shape_of="y"),
```

Comment:
I don't know nitty we want to be about `typing.Dict` vs `Dict` but there are a few of these left

Reply:
I've made it this far, might as well try to go all the way...